api/python/cellxgene_census/tests/test_get_anndata.py

```diff
@@ -195,3 +193,21 @@ def test_get_anndata_obsm_two_layers(census: soma.Collection, obsm_layers: List[
     for obsm_layer in obsm_layers:
         assert obsm_layer in ad.obsm.keys()
         assert ad.obsm[obsm_layer].shape[0] == 100
+
+@pytest.mark.live_corpus
+@pytest.mark.parametrize("add_obs_embeddings", [["scvi", "geneformer"]])
+def test_get_anndata_add_obs_embeddings(census: soma.Collection, add_obs_embeddings: List[str]) -> None:
```

Comment:
Test won't pass until we add `embedding_name` to the manifest file, as well as the new embeddings. Also add different organisms.